app/controllers/SocialAuthController.scala

```diff
@@ -40,7 +41,7 @@ class SocialAuthController @Inject() (
             result <- silhouette.env.authenticatorService.embed(token, Redirect(userState.state("redirect")))
           } yield {
 
-            val url = routes.ActivateAccountController.activate(token).absoluteURL()
+            val url = routes.ActivateAccountController.activate(token, redirectURL).absoluteURL()
```

Comment:
This code runs after authenticating right? Should we actually be getting `redirect` off the `state` param that comes back from google, as on line 41?